src/statement_pruner.cpp

```diff
@@ -96,12 +95,15 @@ const IR::P4Program *prune_statements(const IR::P4Program *program,
 
         } else {
             INFO("PASSED");
+
+            INFO("\nReduced by " << measure_pct(program, temp) << " %\n");
```

Comment:
I would remove the newlines here and all the other prints. Makes the printing more compact. I would also merged it with the PASSED statement.